go/libkb/globals.go

```diff
@@ -30,6 +30,14 @@ import (
 
 type ShutdownHook func() error
 
+type LoginHook interface {
+	OnLogin()
```

Comment:
Login currently means the Login engine ran successfully.  

Which means that there is a valid session token (and possibly other things, like unlocked secret keys and newly provisioned device).

For gregord, it only matters that we have a session token.

It is called at exactly the same place as the NotifyRouter call, so perhaps it would be best to consolidate.